src/pages/booklist/containers/BooklistContainer.tsx

```diff
@@ -52,8 +52,10 @@ export function BooklistContainer(): JSX.Element | null {
             </>
           )}
         </Box>
-        {textbooks.status === 'loaded' && (
-          <Text as="i">Amazon's trademark is used under license from Amazon.com, Inc. or its affiliates</Text>
+        {textbooks.status === 'loaded' && textbooks.textbookInfo.length > 0 && (
+          <Box as="footer" px="2" textAlign={{ base: 'center', md: 'left' }}>
+            <Text as="i">Amazon's trademark is used under license from Amazon.com, Inc. or its affiliates</Text>
+          </Box>
         )}
```

Comment:
I never noticed this but why is the footer conditional on having results?

Reply:
When the spinner is there it was kinda glitching out and jumping around. Figured since the amazon trademark is only there if data is loaded I'd just make it optional